bip-0039/bip-0039-wordlists.md

```diff
@@ -76,7 +76,7 @@ Words chosen using the following rules:
 6. No plural words.
 7. No words that remind negative/sad/bad things.
 8. If both female/male words are available, choose male version.
-9. No words with double vocals (like: lineetta).
+9. No words with double vowels (like: lineetta).
```

Comment:
Yes, per https://translate.google.com/?sl=en&tl=it&text=vowel&op=translate, "vowel" in English appears to be "vocale" in Italian, so this fix looks correct.